src/rgw/rgw_rest_pubsub.cc

```diff
@@ -396,16 +396,16 @@ class RGWPSCreateTopicOp : public RGWOp {
 
 void RGWPSCreateTopicOp::execute(optional_yield y) {
   // master request will replicate the topic creation.
-  if (!driver->is_meta_master()) {
-    op_ret = rgw_forward_request_to_master(
-        this, *s->penv.site, s->owner.id, &bl_post_body, nullptr, s->info, y);
-    if (op_ret < 0) {
-      ldpp_dout(this, 4)
-          << "CreateTopic forward_request_to_master returned ret = " << op_ret
-          << dendl;
-      return;
-    }
-  }
+//  if (!driver->is_meta_master()) {
+//    op_ret = rgw_forward_request_to_master(
+//        this, *s->penv.site, s->owner.id, &bl_post_body, nullptr, s->info, y);
+//    if (op_ret < 0) {
+//      ldpp_dout(this, 4)
+//          << "CreateTopic forward_request_to_master returned ret = " << op_ret
+//          << dendl;
+//      return;
+//    }
+// }
```

Comment:
https://github.com/ceph/ceph/pull/59980 looks like a more complete version of this commit

Reply:
Ah sorry, was meant to be a draft PR. Will commit a more complete fix to https://tracker.ceph.com/issues/68104 soon. 

Was thinking of adding another feature like how notification_v2 was added to disable multisite replication and checking for that, or adding a parameter to the topic creation HTTP request and checking for that?